cromwell.examples.conf

```diff
@@ -74,6 +74,26 @@ system {
 
     #object = 128000
   }
+
+  abort {
+    # These are the default values in Cromwell, in most circumstances there should not be a need to change them.
+
+    # How frequently Cromwell should scan for aborts.
+    scan-frequency: 30 seconds
+
+    # The cache of in-progress aborts. Cromwell will add entries to this cache once a WorkflowActor has been messaged to abort.
+    # If on the next scan an 'Aborting' status is found for a workflow that has an entry in this cache, Cromwell will not ask
```

Comment:
This feature kind of confuses me...

This description makes it sound like you really want to remember everything that's in the set, rather than cache the top n results.

What if an abort takes more than 20 minutes? Or what if we have 100,001 aborts going concurrenty? Is the `WorkflowActor` abort logic good enough to handle multiple abort messages? Is this just here because we want to avoid multiple messages as much as possible?

Reply:
Ex post mergeo: the previous system did not try to shield WAs from duplicate abort requests, this is just a nicety. 🙂